tools/src/debug/bin/kill-process.ts

```diff
@@ -0,0 +1,32 @@
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { PID, killProcesses } from '../utils';
+import { KillProcessesBinOptions } from './types';
+
+const { commandMatch, pid, verbose, force } = yargs(hideBin(process.argv))
+  .version(false)
+  .options({
+    force: { type: 'boolean', default: false },
+    verbose: { type: 'boolean' },
+    pid: { type: 'string', array: true, default: [] },
+    commandMatch: { type: 'string', array: true, default: [] },
+  })
+  .coerce('commandMatch', (commandMatch: string[]) =>
+    commandMatch.flatMap(p => p.split(',')).filter(p => p !== ''),
+  )
+  .coerce('pid', (pid: string[]) =>
+    pid.flatMap(p => p.split(',')).filter(p => p !== ''),
+  ).argv as Omit<KillProcessesBinOptions, 'pid' | 'commandMatch'> & {
+  pid: PID[];
+  commandMatch: string[];
+};
+verbose && commandMatch && console.log(`Command Filter: ${commandMatch}`);
+verbose && pid != null && console.log(`PID Filter: ${pid.join(', ')}`);
```

Comment:
Please use control-flow blocks for executing statements with side-efffects, it's more semantic than evaluation it as an expression.

```suggestion
if (verbose && commandMatch) {
  console.log(`Command Filter: ${commandMatch}`);
}
if (verbose && pid != null) {
  console.log(`PID Filter: ${pid.join(', ')}`);
}
```

Or:

```suggestion
if (verbose) {
  if (commandMatch) {
    console.log(`Command Filter: ${commandMatch}`);
  }
  if (pid != null) {
    console.log(`PID Filter: ${pid.join(', ')}`);
  }
}
```